### PR TITLE
{data}[gompi/2021b] HDF5 v1.10.8

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

Depends on
- [ ] https://github.com/easybuilders/easybuild-easyblocks/pull/2670

### CODE REVIEW

**Comment by lexming** [comment]
This version of HDF5 disabled some binaries due to CVEs. It will need an update of the `hdf5` easyblock.

**Comment by sassy-crick** [comment]
@lexming I guess these are the ones I tipped over when I tried to install it. Are you going to update the `hdf5` easyblock?

**Comment by lexming** [comment]
@sassy-crick yes, I already had an update for the easyblock (I was dealing with this same installation yesterday :smile: ). Waiting for a failed test to hit this PR...

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node301.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/0ba7229ad61968c025ffc75bca2204f5 for a full test report.

**Comment by sassy-crick** [comment]
If you want to do the EC as well as it might be easier, I am happy to close this one. :-)
Up to you.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node301.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c17a4147a3a8319d02ae805b2449ac1a for a full test report.

File: easybuild/easyconfigs/h/HDF5/HDF5-1.10.8-gompi-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['66ec544b195a4cb9f6ffed034fd82e52429d6112747c2996ab69853f606e546b']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Szip', '2.1.1'),
+]
+
+configopts = "--enable-hltools"  # needed to build HL-tools: gif2h5 gif2h5 and h5watch
```
Comment (lexming) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.10.8-gompi-2021b.eb:24]: The vulnerable tools should be left disabled by default
```suggestion
# needed to build HL-tools: gif2h5 gif2h5 and h5watch
# beware that gif tools are affected by multiple CVEs: CVE-2018-17433, CVE-2018-17436, CVE-2020-10809
# configopts = "--enable-hltools" 

```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
@sassy-crick No need to close this PR, it justs need a minor change

**Comment by sassy-crick** [comment]
That looks good to me. I cannot test the file as I don't have the updated EasyBlock. Let me know how it goes.

**Comment by lexming** [comment]
@sassy-crick you can test the `hdf5` easyblock from the PR with `eb --include-easyblocks-from-pr=2670`

**Comment by lexming** [comment]
please test @ generoso
EB_ARGS="--include-easyblocks-from-pr=2670"

**Comment by lexming** [comment]
@boegelbot: please test @ generoso
EB_ARGS="--include-easyblocks-from-pr=2670"

**Comment by sassy-crick** [comment]
> @sassy-crick you can test the `hdf5` easyblock from the PR with `eb --include-easyblocks-from-pr=2670`
I know but I did not know the number of the EasyBlock PR. :-)

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @sassy-crick !

**Pull request merged**